### ISSUE TITLE
Mismatch between PDF guide and sample-project

### ISSUE THREAD

**Issue by jdonboch** [aws-quickstart/quickstart-trek10-serverless-enterprise-cicd#12]
I am following the quickstart guide PDF here:
https://aws-quickstart.s3.amazonaws.com/quickstart-trek10-serverless-enterprise-cicd/doc/serverless-cicd-for-the-enterprise-on-the-aws-cloud.pdf

In Step 4, Item 3, it states: 
> In the GitHub repository for this Quick Start, open the scripts/git-setup.sh file and
> make sure that the variables STACK_NAME and LOCAL_REPO_FOLDER are set correctly.

When I opened this file up, STACK_NAME and LOCAL_REPO_FOLDER are not defined anywhere in this file.  After poking around more, they are defined int he `scripts/deployment-variables.sh` which is sourced by the `git-setup.sh` file.  There also seems to be much more than just those two variables that need to be set (i.e dev/prod account numbers, s3 bucket name).

Am I looking at an outdated version of the quickstart guide?

**Comment by jdonboch** [comment]
Seems like this change was introduced on June 3rd.

**Comment by jaymccon** [comment]
cc @awarzon @jayyeras

**Comment by awarzon** [comment]
Right, this change https://github.com/aws-quickstart/quickstart-trek10-serverless-enterprise-cicd/pull/6 post-dates the PDF. @jayyeras can we update the PDF for the new setup script contributed in https://github.com/aws-quickstart/quickstart-trek10-serverless-enterprise-cicd/pull/6 ?

**Comment by jayyeras** [comment]
> Right, this change #6 post-dates the PDF. @jayyeras can we update the PDF for the new setup script contributed in #6 ?

@awarzon please submit an update copy of the deployment guide to myself and @jaymccon for us to send to our tech writers. thanks!

**Comment by awarzon** [comment]
@jayyeras we have submitted an update to the guide. Thanks.
